### PR TITLE
Workout listall feature

### PR DESCRIPTION
**Pull request by emilysim00**
List all the workouts in the workoutList at one shot

### CODE REVIEW

File: src/main/java/data/workouts/WorkoutList.java [hunk 1]
```diff
@@ -210,6 +210,22 @@ public void listWorkout() {
...
+            Workout workoutObject = getWorkoutFromIndexNum(index + 1);
+            System.out.println(index + 1 + ". " + workoutObject.toString());
+            index += 1;
+            assert (index <= workoutsDisplayList.size());
+        }
+        System.out.println("Showed all workouts in list");
+    }
+
     /**
      * Prints workouts stored in workout list.
```
Comment (Musfirahe0556596) [inline src/main/java/data/workouts/WorkoutList.java:230]: Perhaps could explain a bit further on this JavaDoc. E.g. prints workout in 20 workouts per page.

**Review by Musfirahe0556596 — APPROVED** [review]
Looks great!

**Review by TianaiYan — COMMENTED** [review]
Overall look nice!

File: src/main/java/data/workouts/WorkoutList.java (hunk 1 above)
Comment (emilysim00) [reply src/main/java/data/workouts/WorkoutList.java:230]: Can you elaborate further? I do not quite understand what you mean? This listall feature will display all workouts in the workoutlist and the list feature will display at most 10 workouts at a time. [advance feature]

File: src/main/java/data/workouts/WorkoutList.java [hunk 2]
```diff
@@ -210,6 +210,22 @@ public void listWorkout() {
...
+    /**
+     * Prints all workouts stored in workout list at once.
+     */
+    public void listAllWorkout() {
+        int index = 0;
+        System.out.println("Showing workouts " + (index + 1) + "-" + workoutsDisplayList.size()
+                + " of " + workoutsDisplayList.size() + ":");
+        while (index < workoutsDisplayList.size()) {
+            Workout workoutObject = getWorkoutFromIndexNum(index + 1);
+            System.out.println(index + 1 + ". " + workoutObject.toString());
```
[the remark this answers is not in the document]
Comment (emilysim00) [reply src/main/java/data/workouts/WorkoutList.java:222]: The index is changing every time it loop. As the index+1 is use as the number formatting during printing

**Review by a1021492980 — APPROVED** [review]
Looks good!

**Pull request merged**